pkg/plugins/pfs/pfs.go

```diff
@@ -232,6 +232,41 @@ func ParsePluginFS(fsys fs.FS, rt *thema.Runtime) (ParsedPlugin, error) {
 	return pp, nil
 }
 
+func LoadComposableKindDef(fsys fs.FS, rt *thema.Runtime, defpath string) (kindsys.Def[kindsys.ComposableProperties], error) {
```

Comment:
It's pretty significant to expose a new one of these loaders. This'll definitely need to move as we move pfs around more generally.

Seems OK for now, though.

Would prefer to see a comment on it!